src/transformer_util.ts

```diff
@@ -685,3 +685,14 @@ export function isTypeNodeKind(kind: ts.SyntaxKind) {
       kind === ts.SyntaxKind.UndefinedKeyword || kind === ts.SyntaxKind.NullKeyword ||
       kind === ts.SyntaxKind.NeverKeyword || kind === ts.SyntaxKind.ExpressionWithTypeArguments;
 }
+
+/**
+ * Creates a string literal that uses single quotes. Purely cosmetic, but increases fidelity to the
+ * existing test suite.
```

Comment:
Given that you're rebaselining the tests below, is it worth reducing use of this?

Reply:
It's nice to reduce the diff as we port the code base (even if just during development).  We could reconsider once we're done here.